src/lib/swapper/swappers/ThorchainSwapper/utils/getL1quote.ts

```diff
@@ -128,10 +128,7 @@ export const getL1quote = async (
     return {
       source,
       quote,
-      // don't take affiliate fee into account, this will be displayed as a separate line item
-      expectedAmountOutThorBaseUnit: bnOrZero(quote.expected_amount_out)
-        .plus(bnOrZero(quote.fees.affiliate))
-        .toFixed(),
+      expectedAmountOutThorBaseUnit: bnOrZero(quote.expected_amount_out).toFixed(),
```

Comment:
dont add the affiliate fee to the receive amount because the user will not receive that amount.

Reply:
Left side was obviously wrong from the start, we were taking `expectedAmountOut` and adding `expectedAmountOut + affiliate fees`. `expectedAmountOut` should be === `expected_amount_out`

Reply:
I was a bit confused about this PR, but looking at the history of this line, and the comment above it, I think the [comment](https://github.com/shapeshift/web/pull/5748/files#r1412474585) conveyed the intent, but the [code](https://github.com/shapeshift/web/pull/5748/files#diff-7c6ea7dd104063ccb4b2272b34e9cbe84f738c52b106cb3597368d29e79a53d8R117) did not match - and we missed that in review.

So this now reflects our understanding of the `expected_amount_out` value that we receive.